docs/source/user-guide/tabular/dataset_object.rst

```diff
@@ -155,13 +155,11 @@ will run on the columns to determine which are classified as categorical:
 #. If the ``pd.dtypes`` of any of the existing columns is ``category`` then all of the columns that are of type ``category`` 
    will be  considered categorical (and only them).
 
-#. Otherwise, a heuristic is used for deducting the type. Each column for which at least one of the following conditions is met is considered categorical:
+#. Otherwise, a heuristic is used for deducting the type. In order for a column to be inferred as categorical two condition must be met:
```

Comment:
```suggestion
#. Otherwise, a heuristic is used for deducting the type. In order for a column to be inferred as categorical, the following two conditions must be met:
```